arcadia-users-group/20220822_intro_to_markdown_syntax.md

```diff
@@ -0,0 +1,156 @@
+# Introduction to markdown syntax
+
+Markdown is a lightweight markup language.
+A markup language is a "text-encoding system consisting of a set of symbols inserted in a text document to control its structure, formatting, or the relationship between its parts" ([source](https://en.wikipedia.org/wiki/Markup_language)).
+Other common markup languages include LaTeX and HTML.
+Markdown was created to enable people to easily read and write plain text format.
+
+Markdown (and other markup languages) can be automatically rendered to control the display format of a document.
+It's often used in formatting for things like websites & blogs and in computational notebooks (jupyter & RMarkdown).
```

Comment:
I'd also add a note about the fact that Notion uses Markdown and this is useful for that as well